plasmapy/physics/parameters.py

```diff
@@ -524,11 +520,177 @@ def kappa_thermal_speed(T, kappa, particle="e", method="most_probable"):
         raise ValueError("Method {method} not supported in thermal_speed")
 
 
+def particle_params(particle):
+    """Helper function to extract data from particle strings"""
+    try:
+        m = atomic.ion_mass(particle)
+        try:
+            Z = atomic.charge_state(particle)
+        except ValueError:
+            Z = 1
+        Z = abs(Z)
+    except Exception:
+        raise ValueError(f"Invalid particle {particle}")
+    return (m, Z)
+
+
+@utils.check_quantity({
+    'T_e': {'units': units.K, 'can_be_negative': False},
+    'n_e': {'units': units.m**-3, 'can_be_negative': False}
+})
+def electron_ion_collision_rate(T_e, n_e, ion_particle,
+                                coulomb_log=None, V=None):
+    r"""
+    momentum relaxation electron-ion collision rate
+
+    From Callen Chapter 2, http://homepages.cae.wisc.edu/~callen/chap2.pdf,
+    equations (2.17) and (2.120)
+
+    Considering a Maxwellian distribution of "test" electrons colliding with
+    a Maxwellian distribution of "field" ions.
+
+    This result is an electron momentum relaxation rate, and is used in many
+    classical transport expressions. It is equivalent to:
+    * 1/tau_e from ref [1]_ eqn (1) pp. #,
+    * 1/tau_e from ref [2]_ eqn (1) pp. #,
+    * nu_e\i_S from ref [2]_ eqn (1) pp. #,
+
+    Parameters
+    ----------
+
+    T_e : ~astropy.units.Quantity
+        The electron temperature of the Maxwellian test electrons
+
+    n_e : ~astropy.units.Quantity
+        The number density of the Maxwellian test electrons
+
+    ion_particle: string
+        String signifying a particle type of the field ions, including charge
+        state information.
+
+    coulomb_log : float or dimensionless ~astropy.units.Quantity, optional
+        option to specify a Coulomb logarithm of the electrons on the ions.
+        If not specified, the Coulomb log will is calculated using the
+        .transport/Coulomb_logarithm function.
+
+    References
+    ----------
+    .. [1] Braginskii
+
+    .. [2] Formulary
+
+    """
+    from plasmapy.physics.transport import Coulomb_logarithm
+    T_e = T_e.to(units.K, equivalencies=units.temperature_energy())
+    if coulomb_log is not None:
+        coulomb_log_val = coulomb_log
+    else:
+        particles = ['e', ion_particle]
+        coulomb_log_val = Coulomb_logarithm(T_e, n_e, particles, V)
+    Z_i = atomic.charge_state(ion_particle)
```

Comment:
I believe average ionization is already merged so you should rebase and add that functionality here.